src/LostAndFound/GameManager.cs

```diff
@@ -23,10 +23,35 @@ private static void PrintIntro()
             TimedEvents.TimedWrite(introText, TimedEvents.WriteSpeed.Slow);
         }
 
-        private static void ManageGameActions()
+        private static void ManageMenuActions()
         {
             PrintMainMenu();
             MenuChoice userMenuChoice = GetUserMenuChoice();
+
+            if (userMenuChoice == MenuChoice.Start)
```

Comment:
You are missing the gameplay loop. After selecting any action, the game is exiting